frontend/src/App.tsx

```diff
@@ -17,7 +17,7 @@ import Frame from './features/frame/Frame';
 import ComparisonPage from './pages/comparisons/Comparison';
 import PublicRoute from './features/login/PublicRoute';
 import PrivateRoute from './features/login/PrivateRoute';
-import VideoCardPage from './pages/videos/VideoCard';
+import VideoAnalysisPage from './pages/videos/VideoAnalysis';
```

Comment:
As `recharts` is quite a big dependency (https://bundlephobia.com/package/recharts@2.1.8), and is only used on this single route, it would make sense to import this component lazily (it will be built in separate .js chunks, which won't be downloaded by most visitors).

```ts
const VideoAnalysisPage = React.lazy(
  () => import('./pages/videos/VideoAnalysis')
);
```

Also see https://reactjs.org/docs/code-splitting.html#route-based-code-splitting for more details.